pkg/sass-parser/lib/src/expression/number.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import * as postcss from 'postcss';
+
+import {LazySource} from '../lazy-source';
+import type * as sassInternal from '../sass-internal';
+import * as utils from '../utils';
+import {Expression} from '.';
+
+/**
+ * The initializer properties for {@link NumberExpression}.
+ *
+ * @category Expression
+ */
+export interface NumberExpressionProps {
+  value: number;
+  unit?: string;
+  raws?: NumberExpressionRaws;
+}
+
+/**
+ * Raws indicating how to precisely serialize a {@link NumberExpression}.
+ *
+ * @category Expression
+ */
+export interface NumberExpressionRaws {
+  /**
+   * The raw string representation of the number.
+   *
+   * Numbers can be represented with or without leading and trailing zeroes, and
+   * use scientific notation. For example, the following number representations
+   * have the same value: `1e3`, `1000`, `01000.0`.
+   */
+  // TODO: Replace with RawWithValue<T> when #2389 lands.
+  number?: {raw?: string; value?: number};
```

Comment:
I'd name this `value` to match the property on `NumberExpression`.

Also, `raw` and the inner `value` should be required.

Reply:
Yeah, I wasn't too sure and I ended up changing it. I think it should be `value` as you suggest to match `NumberExpression` (both here and in the sass_api), but I was getting confused with `raws.value.value` so I changed it. I'll change it back to `value` and we can revisit this later if it's too confusing.